File: photography/api/hireapi.php
<?php
//header ('Content-type: application/json');
//$method = $_SERVER['REQUEST_METHOD'];
require_once "../connectdb.php";
//mysqli_select_db($conect,"acswesm5_allpo_db");

	$hname = $_POST['hname'];
	$hcontact =$_POST['hcontact'];
	$hlocation = $_POST['hlocation'];
	$hetype = $_POST['hetype'];
	$hmsg = $_POST['hmsg'];
	$userid = $_POST['userid'];
	$hdate = $_POST['hdate'];
	$pgname = $_POST['pgname'];
	$pgmail = $_POST['pgmail'];
	
	$query="INSERT INTO hiretbl(hname, hcontact, hlocation, hetype, hmsg, userid, hdate, pgname, pgmail) VALUES('$hname','$hcontact','$hlocation','$hetype','$hmsg','$userid','$hdate','$pgname','$pgmail')";
	mysqli_query($conect,$query);
	echo json_encode(array('response'=>"Inserted Succesfully"));


?>

File: Professional Photographers/app/src/main/java/com/jovialway/professionalphotographers/hirecontact/HireApi.java
package com.jovialway.professionalphotographers.hirecontact;

import retrofit2.Call;
import retrofit2.http.Field;
import retrofit2.http.FormUrlEncoded;
import retrofit2.http.POST;

public interface HireApi {
    @FormUrlEncoded
    @POST("photography/api/hireapi.php")
    Call<HModel> addhiredata(@Field("hname") String hname, @Field("hcontact") String hcontact, @Field("hlocation") String hlocation,@Field("hetype") String hetype,@Field("hmsg") String hmsg, @Field("userid") String userid, @Field("hdate") String hdate, @Field("pgname") String pgname, @Field("pgmail") String pgmail);
}
